package.json

```diff
@@ -6,32 +6,35 @@
   "author": "The KeystoneJS Development Team",
   "license": "MIT",
   "scripts": {
-    "coverage": "DISABLE_LOGGING=true NODE_ENV=test jest --maxWorkers=1 --coverage",
+    "demo-projects": "yarn --cwd ./demo-projects",
+    "test-projects": "yarn --cwd ./test-projects",
```

Comment:
If I run `yarn test-projects`, what happens? Does it do the equivalent of `cd test-projects && yarn install`?

Reply:
It is the equivalent of `cd test-projects && yarn` so I believe that is the same result

Reply:
And does that actually do anything (given that directory's `package.json` doesn't have any `dependencies`)? (sorry, I'd test myself, but I'm neck deep in another branch right now)

Reply:
No. It is true that it is a little annoying for new users.

We can fix _allthethings_ but doing
```json
"website:": "yarn --cwd website",
"demo:": "yarn --cwd demo-projects",
"demo:": "yarn --cwd test-projects",
```

Then we have `yarn test: start`  and `yarn test: basic: start` so it is more obvious that `yarn test:` is a oops.

However I can see that in docs the space may be considered a type-o. Its a case of chose-the-lesser-confusion. 🤔

Reply:
Thinking about it more; I don't ever expect a user to _run_ the test projects. They're designed to be run by CI only really.

So perhaps the naming inconsistency is ok:

```
$ yarn website start
$ yarn demo blog start
$ yarn test-project basic start # only run by CI
```

Reply:
Or... we don't even expose the test projects to be run like that, which makes even more sense to me:

```
$ yarn website start
$ yarn demo blog start
$ cd test-projects/basic && yarn start # a user is discouraged from doing this
```